src/Widgets/NavigationWidget/SubComponents/LanguageSwitch.tsx

```diff
@@ -0,0 +1,87 @@
+import { NavDropdown } from 'react-bootstrap'
+import {
+  ImageTag,
+  InPlaceEditingOff,
+  LinkTag,
+  Obj,
+  connect,
+  currentPage,
+  currentSiteId,
+} from 'scrivito'
+import { HomepageInstance } from '../../../Objs/Homepage/HomepageObjClass'
+
+export const LanguageSwitch = connect(function LanguageSwitch() {
```

Comment:
High-level feedback: It would be nice, if the LanguageSwitch disappears, if only one site exists (so it is easy to convert the "multi-language portal" to a "single language portal").

E.g. in the working-copy https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://dave-language-switch.scrivito-portal-app.pages.dev/en?_scrivito_workspace_id=j539ba0e70061273&_scrivito_display_mode=view

Reply:
* [x] f839952